tools/importer-rest-api-specs/components/parser/swagger_tags_test.go

```diff
@@ -44,3 +44,69 @@ func TestParsingOperationsUsingTheSameSwaggerTagInDifferentCasings(t *testing.T)
 		}
 	}
 }
+
+func TestParsingOperationsOnResources(t *testing.T) {
+	result, err := ParseSwaggerFileForTesting(t, "operations_on_resources.json")
+	if err != nil {
+		t.Fatalf("parsing: %+v", err)
+	}
+	if result == nil {
+		t.Fatal("result was nil")
+	}
+	if len(result.Resources) != 2 {
+		t.Fatalf("expected 2 resource but got %d", len(result.Resources))
+	}
+
+	resource, ok := result.Resources["Hello"]
+	if !ok {
+		t.Fatal("the Resource 'Hello' was not found")
+	}
+
+	// sanity checking
+	if len(resource.Constants) != 0 {
+		t.Fatalf("expected 0 constants but got %d", len(resource.Constants))
+	}
+	if len(resource.Models) != 1 {
+		t.Fatalf("expected 1 model but got %d", len(resource.Models))
+	}
+	if len(resource.Operations) != 3 {
+		t.Fatalf("expected 4 Operations but got %d", len(resource.Operations))
```

Comment:
3 or 4?

Reply:
3, because the 4th operation belongs to the 2nd resource.